scripts/build-openapi.js

```diff
@@ -1,113 +1,51 @@
 #!/usr/bin/env node
 
 const fs = require("fs")
-const OAS = require("oas-normalize")
-const swaggerInline = require("swagger-inline")
-const { exec } = require("child_process");
+const path = require("path")
+const { exec } = require("child_process")
+const yaml = require("js-yaml")
 
-const isDryRun = process.argv.indexOf('--dry-run') !== -1;
+const run = () => {
+  for (const apiType of ["store", "admin"]) {
+    const inputJsonFile = path.resolve(
+      __dirname,
+      "../",
```

Comment:
suggestion: Same as previously, putting `../` into a const named `basePath` ? to be used in the different places

Reply:
Done